### DISCUSSION TITLE
Airflow dag sensors creation

### DISCUSSION THREAD

**Discussion by shwar123** [apache/airflow#30942] [Q&A]
Hi Team,

I have a requirement like below. 

I have 2 azure adf pipelines like one is parent and other one is child 

I need to shedule both in airflow using sensors once my parent executed successfully then only my chaild need to execute.

Can anyone of you please help me with the solution at earliest.

Thank you!

Eshwar

**Comment by pankajastro** [comment]
Hi @shwar123 are you getting some error or any specific issue you are facing? I think this should be possible 

- check the ADF status using `AzureDataFactoryPipelineRunStatusSensor` 
- you can execute ADF pipeline using `AzureDataFactoryRunPipelineOperator`
- and you can define [task dependency](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html) in airflow using `>>` like `pipeline1 >> pipeline2` run pipeline2 if one succeeded.
